number_of_robots: 1

quadrotor_name: dragonfly11
mass: 0.250 # redundant of robot_mass
robot_mass: 0.250
payload_mass: 0.037
cable_length: 0.55
my_index: 0

attachment:
  rho_0: {x: 0.000, y: 0.000, z: 0.000}
